src/pymodaq/pid/pid_controller.py

```diff
@@ -592,7 +596,7 @@ def start_PID(self, sync_detectors=True, sync_acts=False):
                     self.outputs = [pid.setpoint for pid in self.pids]
 
                 dt = time.perf_counter() - self.current_time
-                self.outputs_to_actuators = self.model_class.convert_output(self.outputs, dt, stab=True)
+                self.outputs_to_actuators = self.model_class.convert_output(self.outputs)
```

Comment:
You should leave it, why removing these, there are usefull!

Reply:
see the PIDModelBoiler example

Reply:
I put them back!